diff --git a/lib/Makefile.am b/lib/Makefile.am
index 71ca1c3..8867b30 100644
--- a/lib/Makefile.am
+++ b/lib/Makefile.am
@@ -195,7 +195,7 @@ libgnutls_xssl_la_LDFLAGS = -no-undefined -export-symbols-regex '^xssl.*'
 
 lib_LTLIBRARIES += libgnutls-xssl.la
 
-libgnutls_xssl_la_SOURCES = xssl.c xssl_getline.c xssl.h
+libgnutls_xssl_la_SOURCES = xssl.c xssl_getline.c xssl.h vasprintf.c
 
 libgnutls_xssl_la_LIBADD = ../gl/libgnu.la libgnutls.la
 
