diff --git a/src/gui/Qt5GuiConfigExtras.cmake.in b/src/gui/Qt5GuiConfigExtras.cmake.in
index d734e56..7166e2e 100644
--- a/src/gui/Qt5GuiConfigExtras.cmake.in
+++ b/src/gui/Qt5GuiConfigExtras.cmake.in
@@ -98,9 +98,6 @@ macro(_qt5gui_find_extra_libs Name Libs LibDir IncDirs)
             find_library(Qt5Gui_${_cmake_lib_name}_LIBRARY ${_lib}
 !!IF !isEmpty(CROSS_COMPILE)
                 PATHS \"${LibDir}\"
-!!IF !mac
-                NO_DEFAULT_PATH
-!!ENDIF
 !!ENDIF
             )
 !!IF mac
@@ -135,9 +132,6 @@ macro(_qt5gui_find_extra_libs Name Libs LibDir IncDirs)
 
                 find_library(Qt5Gui_${_cmake_lib_name}_LIBRARY_DEBUG ${_lib}d
                     PATHS \"${LibDir}\"
-!!IF !mac
-                    NO_DEFAULT_PATH
-!!ENDIF
                 )
                 if (Qt5Gui_${_cmake_lib_name}_LIBRARY_DEBUG)
                     set_property(TARGET Qt5::Gui_${_cmake_lib_name} APPEND PROPERTY IMPORTED_CONFIGURATIONS DEBUG)
